BookingsApi/BookingsApi.Contract/Responses/HearingAllocationsResponse.cs

```diff
@@ -0,0 +1,49 @@
+using System;
```

Comment:
If we are going to refactor the response for the work allocation we should refactor also the search response in HearingsController.SearchForAllocationHearings. in this way we have only one type of response in the same component

Reply:
I have, I think you missed that I have changed the return type and tests for the search endpoint too

Reply:
sorry just saw it.